public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -53,6 +54,8 @@ interface State {
   // If the datasource supports custom metadata
   metaDS?: DataSourceApi;
 
+  stats: { requestTime: number; queries: number; dataSources: number; processingTime: number };
```

Comment:
not seeing where this is used anywhere, it's calculated in the stats render tab now , any reason for adding this back to state ?

Reply:
Oh, must have been a merge error. Reverting